### PR TITLE
Add collection filter

### PR DESCRIPTION
**Pull request by BobWithHair**
Simple feature to add a filter on the collection tag with an environment variable.

Related to #21

### CODE REVIEW

File: src/api/plex.ts [hunk 1]
```diff
@@ -112,8 +113,20 @@ export const allMovies = (async () => {
     }
 
     const libraryData: PlexMediaContainer<PlexVideo> = await req.json()
+    let metadata = libraryData.MediaContainer.Metadata
+
+    if (COLLECTION_FILTER !== '') {
+      const collectionFilter = COLLECTION_FILTER.split(',')
+      metadata = metadata.filter(x => {
```
Comment (LukeChannings) [inline src/api/plex.ts:120]: Can you rename `x` either to `_`, or something longer and more descriptive?

Other than that this MR LGTM.

**Review by LukeChannings — CHANGES_REQUESTED** [review]

**Comment by LukeChannings** [comment]
Also, note that there is work happening on this in the 'dev` branch, but I'm happy to keep releasing changes to 1.x as long as people want to submit them. Thanks!

**Comment by BobWithHair** [comment]
> 
> 
> Also, note that there is work happening on this in the 'dev` branch, but I'm happy to keep releasing changes to 1.x as long as people want to submit them. Thanks!

Yeah I noticed all the work you put in and it's awesome, I see this MR as a quick feature while waiting for the dev branch to be merged.

File: src/api/plex.ts (hunk 1 above)
Comment (BobWithHair) [reply src/api/plex.ts]: I renamed it, LMK if it is ok

**Review by LukeChannings — APPROVED** [review]
LGTM!

**Pull request merged**